src/Spryk/Model/Spryk/Filter/HttpResponseCodeToConstantNameFilter.php

```diff
@@ -0,0 +1,64 @@
+<?php
+
+/**
+ * Copyright © 2016-present Spryker Systems GmbH. All rights reserved.
+ * Use of this software requires acceptance of the Evaluation License Agreement. See LICENSE file.
+ */
+
+namespace SprykerSdk\Spryk\Model\Spryk\Filter;
+
+use ReflectionClass;
+use RuntimeException;
+use Symfony\Component\HttpFoundation\Response;
+
+class HttpResponseCodeToConstantNameFilter implements FilterInterface
+{
+    /**
+     * @var array<string, mixed>
+     */
+    protected $responseClassConstants = [];
```

Comment:
property type is missing. Shouldn't this be a static one for performance reasons?